v1.4.1.md

```diff
@@ -0,0 +1,157 @@
+- [BUG] Failed to display storage class key/value pairs when edit config on Rancher managed Harvester [#7230](https://github.com/harvester/harvester/issues/7230)
+- [backport v1.4] [BUG] rancher/rancher-agent:v2.9.2 image missing in the Harvester ISO [#7225](https://github.com/harvester/harvester/issues/7225)
+- [BUG] Unable to display vm serial console window content on Rancher managed Harvester  [#7220](https://github.com/harvester/harvester/issues/7220)
+- [backport v1.4] [BUG] Harvester VM Metrics Are Not Accurate [#7217](https://github.com/harvester/harvester/issues/7217)
+- [backport v1.4] [BUG] Unable to configure the storage network setting to associate with the default mgmt cluster network [#7213](https://github.com/harvester/harvester/issues/7213)
+- [BUG] Failed to enable storage network on Rancher managed Harvester, can't accept value input on VLAN ID field [#7188](https://github.com/harvester/harvester/issues/7188)
+- [BUG] Failed to create vm template on Rancher managed Harvester [#7183](https://github.com/harvester/harvester/issues/7183)
+- [BUG] Click suspend/resume again have no response on the Rancher managed Harvester [#7174](https://github.com/harvester/harvester/issues/7174)
+- [BUG] Failed to add label to image on the Rancher managed Harvester, display 0% progress  [#7172](https://github.com/harvester/harvester/issues/7172)
+- [backport v1.4] [BUG] [UI] Generate Support Bundle button not working in Rancher/Harvester [#7154](https://github.com/harvester/harvester/issues/7154)
+- [backport v1.4] [BUG] VM encounter the IOError with hotplug volume when the owner node enter the maintenance mode [#7152](https://github.com/harvester/harvester/issues/7152)
+- [backport v1.4] [BUG] Harvester UI shows incorrect number of vGPU allocated to the VM [#7070](https://github.com/harvester/harvester/issues/7070)
+- [backport v1.4] [BUG] API Key table is not refreshed with latest data when deleting API key [#7057](https://github.com/harvester/harvester/issues/7057)
+- [backport v1.4] [BUG] Crash when `VirtualMachineTemplateVersion` contains bogus JSON data [#7054](https://github.com/harvester/harvester/issues/7054)
+- [backport v1.4] [BUG]  VM unable to reach Harvester VIP / Node IP after RKE2 restart [#7044](https://github.com/harvester/harvester/issues/7044)
+- [backport v1.4] [BUG] Upgrade  stuck with Pre-draining due to fail to migrate a Running VM with Container Disk [#7009](https://github.com/harvester/harvester/issues/7009)
+
+
+## Known Issues
+
+- [backport v1.4] [BUG] Fail to trigger v1.4.0 to v1.4.1-rc1 upgrade on witness cluster due to managed chart harvester is not ready [#7414](https://github.com/harvester/harvester/issues/7414)
+- [BUG] Fail to trigger v1.4.0 to v1.4.1-rc1 upgrade on witness cluster due to managed chart harvester is not ready [#7375](https://github.com/harvester/harvester/issues/7375)
```

Comment:
Duplicated issues.

Reply:
should we change the target to v1.4.1?

Reply:
Probably -- the text is just lifted from the issue title, but it might be nicer to just say v1.4.0 (without rc1) in the release notes